patches/chrome-browser-ui-views-download-bubble-download_toolbar_button_view.cc.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/chrome/browser/ui/views/download/bubble/download_toolbar_button_view.cc b/chrome/browser/ui/views/download/bubble/download_toolbar_button_view.cc
+index 394cded1d5e83248d5f26b371c2f2f521d760163..1c11f4bc3e7141b524ad46cc2984f74203eac33d 100644
+--- a/chrome/browser/ui/views/download/bubble/download_toolbar_button_view.cc
++++ b/chrome/browser/ui/views/download/bubble/download_toolbar_button_view.cc
+@@ -419,6 +419,8 @@ void DownloadToolbarButtonView::OpenPrimaryDialog() {
+   primary_view_->SetVisible(true);
+   security_view_->SetVisible(false);
+   bubble_delegate_->SetButtons(ui::DIALOG_BUTTON_NONE);
++  // NOTE: Copied from latest upstream in advance. Delete this when it's arrived.
++  bubble_delegate_->SetDefaultButton(ui::DIALOG_BUTTON_NONE);
+   bubble_delegate_->set_margins(GetPrimaryViewMargin());
+   ResizeDialog();
+ }
+@@ -468,6 +470,8 @@ void DownloadToolbarButtonView::CreateBubbleDialogDelegate(
+   bubble_delegate->SetShowTitle(false);
+   bubble_delegate->SetShowCloseButton(false);
+   bubble_delegate->SetButtons(ui::DIALOG_BUTTON_NONE);
++  // NOTE: Copied from latest upstream in advance. Delete this when it's arrived.
```

Comment:
pulling upstream patches is fine, but pls keep them in the original form (don't add comments).
This way rebase team won't have to solve conflicts, patches should be auto-applied as no-ops.

Reply:
ok, removed.